src/app/server/Server.h

```diff
@@ -54,4 +54,4 @@ enum class ResetAdmins
 /**
  * Open the pairing window using default configured parameters.
  */
-CHIP_ERROR OpenDefaultPairingWindow(chip::ResetAdmins resetAdmins);
+CHIP_ERROR OpenDefaultPairingWindow(chip::ResetAdmins resetAdmins, bool onNetwork = false);
```

Comment:
nit: boolean parameters are generally hard to read. Consider some enum value.

what is onNetwork for ?

would it be something like `kMdnsAdvertise` and `kBleAdvertise` ?

Should we error out if asked to ble advertise but NETWORK_LAYER_BLE is not defined?

Reply:
@cecille (and @andy31415),

Before we merge, is this part of the comment resolved?

"Should we error out if asked to ble advertise but NETWORK_LAYER_BLE is not defined?"

I see we have the enum now, but I can't tell from the code whether this other piece is resolved.

Reply:
Ah, sorry - missed this part. Right now this never gets called if BLE isn't defined, so it's not going to cause a regression. But, yes, in general I would like to share more code between the BLE and softAP so I will add this in a new PR.